design-system-docs/src/_component_docs/pagination.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Pagination
+---
+
+<%= render(Shared::ComponentExample.new(:pagination, :default)) %>
+
+## Accessibility requirements
+
+- Pagination is wrapped in a `<nav>` with `role="navigation"` and an appropriate `aria-label` e.g. `aria-label="Pagination navigation"`. You may wish to customise this based on the context it is used in, e.g. "Search pagination"
+- Each link has an appropriate `aria-label` e.g. "Go to first page", or "Go to page 3".
+- The current page has an `aria-label` of `aria-label="Current page, page 5"` with `aria-current="page"`
+
```

Comment:
I noticed when comparing the docs that there used to be a paging-info cads class (I think it's not really a component): https://citizensadvice.github.io/design-system/?path=/docs/components-pagination--example#paging-info. I think it makes sense not to add it here but wondering if we should mention it elsewhere?

Reply:
I removed it for the time being as @davidsauntson and I realised a while back that it's kinda unused and we completely override it in public-website. So maybe even something we can remove. It's literally just a paragraph which is kinda confusing.